mautic/mautic
# New webhook: Company Create/Update Event

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR added new webhook when company is created or updated.


#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create and copy test webhook url https://webhook.site/
3. Go to Mautic > Webhook and create new one with checked Company Create/Update Event

![image](https://user-images.githubusercontent.com/462477/69066686-022eaa80-0a22-11ea-8461-9469e3ac4a52.png)
4. Go to companies section and try create/update companies. 
5. Check If your https://webhook.site/ receive info about your companies create/update actions
